src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableEntityContext.cs

```diff
@@ -415,12 +417,94 @@ string IDurableEntityContext.StartNewOrchestration(string functionName, object i
             return instanceId;
         }
 
+        async Task IDurableEntityContext.DispatchAsync<T>(params object[] constructorParameters)
+        {
+            IDurableEntityContext context = (IDurableEntityContext)this;
+            MethodInfo method = FindMethodForContext<T>(context);
+
+            if (method == null)
+            {
+                if (string.Equals("delete", context.OperationName, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    Entity.Current.DeleteState();
+                    return;
+                }
+                else
+                {
+                    throw new InvalidOperationException($"No operation named '{context.OperationName}' was found.");
+                }
+            }
+
```

Comment:
I think this block could have a small clarification comment. Is it expected that `"delete"` won't be found via `FindMethodForContext` or is that only under some edge-case? Also, do we expect to grow this if-statement with other special operations?

I would be good to add a clarifying comment answering the questions above :) ^

Reply:
I'm not sure. I copied this from the existing extension method code.

My guess is that we automatically have functionality for "Delete" built in, so the customer doesn't need their own implementation.

Reply:
Fair enough. If you don't mind, it might be good to have that in the code itself, but I wont' block for it